survey_api.go

```diff
@@ -70,6 +70,69 @@ func (a *App) alertOnSurveyCompletion(surveyID uint) error {
 	return nil
 }
 
+// ContactRequest represents the data HubSpot needs for a new contact
+type ContactRequest struct {
+	Properties []Property `json:"properties"`
+}
+
+type Property struct {
+	Property string `json:"property"`
+	Value    string `json:"value"`
+}
+
+// createHubSpotContact sends the user's email and other data to HubSpot as a new contact
+func (a *App) createHubSpotContact(email, companyName, userRole string) error {
+	// Define the HubSpot API URL
+	HubSpotAPIURL := "https://api.hubapi.com/contacts/v1/contact"
+
+	// Get the HubSpot API token from the environment
+	HubSpotAPIToken := os.Getenv("HUBSPOT_API_KEY")
+	if HubSpotAPIToken == "" {
+		log.Println("HubSpot API token not set")
+		return fmt.Errorf("HubSpot API token not set")
```

Comment:
Super nit suggestion -- but we're already using the errors package so if you're not going to wrap an error we can create a new one. It will provide us a lot more functionality for diagnosing errors rather than just logging them.

```suggestion
		return errors.New("Hubspot API token not set")
```